src/reducers/app.js

```diff
@@ -33,6 +33,7 @@ const view: Reducer<AppViewState> = (
       return { phase: 'FATAL_ERROR', error: action.error };
     case 'WAITING_FOR_PROFILE_FROM_ADDON':
     case 'WAITING_FOR_PROFILE_FROM_URL':
+    case 'WAITING_FOR_PROFILE_FROM_FILE':
```

Comment:
Is this view introduced from this PR, or did we just miss it?

Reply:
No, this was already existing.